deployments/ui/kagenti-ui.yaml

```diff
@@ -25,6 +25,45 @@ spec:
         - name: kagenti-ui-container
           image: ghcr.io/kagenti/kagenti/ui:latest
           imagePullPolicy: IfNotPresent
+          env:
+            - name: "ENABLE_AUTH"
+              value: "false"
+            - name: "CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  name: "auth" 
```

Comment:
"auth" is not a good name for a secret.  Maybe kagenti-ui-oauth-secret would be better.

This Pod depends on the secret being present.  Check in the secret as part of this PR so that the pod can start.  The values in the secret can be TODOs.  Ideally if the user enables auth it will fail with messages like "Unknown OIDC client ID: TODO update CLIENT_ID in kagenti-ui-oauth-secret".

Reply:
`optional` is good enough; these parameters truly are optional because they are not needed if ENABLE_AUTH is not `true`.